### PR TITLE
Add release notes to versions 4.6.10, 4.8.0.1 and 4.0.18

### PR DESCRIPTION
**Pull request by mscibisz**
Please mind that links to the new binaries are not available yet.

### CODE REVIEW

**Comment by pietrushnic** [comment]
@mscibisz please let me know about the release, so I can publish this information in social media.

**Comment by mscibisz** [comment]
@pietrushnic All links are working. If there is everything OK here, please approve this MR @miczyg1

File: index.html [hunk 1]
```diff
@@ -117,15 +120,107 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
...
+                    <li><a href="http://pcengines.ch/file/apu4_v4.8.0.1.rom.tar.gz">apu4 v4.8.0.1</a></li>
+                    <li><a href="http://pcengines.ch/file/apu5_v4.8.0.1.rom.tar.gz">apu5 v4.8.0.1</a></li>
+                </ul>
+            </div>
+            <div class="source-code-column">
+                <div class="smaller-margin"><p>Source code:</p></div>
+                <ul>
+                    <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.10...v4.8.0.1">coreboot v4.8.0.1</a></li>
+                    <li><a target="_blank" href="https://github.com/pcengines/seabios/compare/rel-1.11.0.4...rel-1.11.0.5">SeaBIOS rel-1.11.0.4</a></li>
+                    <li><a target="_blank" href="https://github.com/pcengines/sortbootorder/compare/v4.6.8...v4.6.9">sortbootorder v4.6.8</a></li>
```
Comment (miczyg1) [inline index.html]: @mscibisz wyświetlony powinien być sortbootorder v4.6.9

File: index.html [hunk 2]
```diff
@@ -117,15 +120,107 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
...
+                    <li><a href="http://pcengines.ch/file/apu3_v4.8.0.1.rom.tar.gz">apu3 v4.8.0.1</a></li>
+                    <li><a href="http://pcengines.ch/file/apu4_v4.8.0.1.rom.tar.gz">apu4 v4.8.0.1</a></li>
+                    <li><a href="http://pcengines.ch/file/apu5_v4.8.0.1.rom.tar.gz">apu5 v4.8.0.1</a></li>
+                </ul>
+            </div>
+            <div class="source-code-column">
+                <div class="smaller-margin"><p>Source code:</p></div>
+                <ul>
+                    <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.10...v4.8.0.1">coreboot v4.8.0.1</a></li>
+                    <li><a target="_blank" href="https://github.com/pcengines/seabios/compare/rel-1.11.0.4...rel-1.11.0.5">SeaBIOS rel-1.11.0.4</a></li>
```
Comment (miczyg1) [inline index.html]: @mscibisz wyświetlony powinien być SeaBIOS rel-1.11.0.5

File: index.html [hunk 3]
```diff
@@ -117,15 +120,107 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
...
+                    <li><a href="http://pcengines.ch/file/apu3_v4.6.10.rom.tar.gz">apu3 v4.6.10</a></li>
+                    <li><a href="http://pcengines.ch/file/apu4_v4.6.10.rom.tar.gz">apu4 v4.6.10</a></li>
+                    <li><a href="http://pcengines.ch/file/apu5_v4.6.10.rom.tar.gz">apu5 v4.6.10</a></li>
+                </ul>
+            </div>
+            <div class="source-code-column">
+                <div class="smaller-margin"><p>Source code:</p></div>
+                <ul>
+                    <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.9...v4.6.10">coreboot v4.6.10</a></li>
+                    <li><a target="_blank" href="https://github.com/pcengines/seabios/compare/rel-1.11.0.4...rel-1.11.0.5">SeaBIOS rel-1.11.0.4</a></li>
```
Comment (miczyg1) [inline index.html]: Napisy SeaBIOS i sortbootorder jak wyżej @mscibisz

**Comment by mscibisz** [comment]
@miczyg1 Fixed

**Pull request merged**